src/main/haskell/language-kore/src/Data/Kore/AST.hs

```diff
@@ -84,11 +85,12 @@ Note that this is very similar to 'SymbolOrAlias'.
 -}
 data Symbol a = Symbol
     { symbolConstructor :: !(Id a)
-    , symbolParams      :: ![Sort a]
+    , symbolParams      :: ![UnifiedSortVariable]
```

Comment:
Why is it not `![Sort a]` but `![UnifiedSortVariable]`?

Reply:
Fixed, thanks.